rest/answering-machine-detection/outgoing-call/outgoing-call-1.6.x.py

```diff
@@ -0,0 +1,14 @@
+# Download the Python helper library from twilio.com/docs/python/install
+from twilio.rest import Client
+
+# Your Account Sid and Auth Token from twilio.com/user/account
+account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
```

Comment:
I'd like to keep the usage of quotes consistent across the snippets. Perhaps you want to take a look at https://stackoverflow.com/questions/56011/single-quotes-vs-double-quotes-in-python. Moreover, I want to take a look at the coding conventions we're using for Python in this particular case.